index.html

```diff
@@ -66,6 +66,17 @@ <h2 class="tittle-header">My Recent Works</h2>
             </div>
             <div class="work-cards">
                 <div class="content-container">  
+                    <div class="card1-container"></div> 
+                    <div class="cards-frame"></div>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Great job working on the pop up so far but I think you missed one important step since in the requirements you should remove the hard coded for the cards from HTML and create them using js like the pop up cards so kindly remove all the cards from the HTML and generate it with JS as required 🔥🤩🔥✨💯✅✅


![image](https://user-images.githubusercontent.com/58553711/225185675-7595b310-f3a7-43bc-89be-0c5316b7c985.png)